tx_handler/migration_tx_handler.go

```diff
@@ -86,13 +87,12 @@ func (h *MigrationTxHandler) ProcessTx(
 	if migrationFn == nil {
 		return r, fmt.Errorf("invalid migration ID %d", tx.ID)
 	}
-
 	migrationCtx := migrations.NewMigrationContext(h.Manager, h.CreateRegistry, state, origin)
-	if err := migrationFn(migrationCtx); err != nil {
+	if err := migrationFn(migrationCtx, tx.Input); err != nil {
 		return r, errors.Wrapf(err, "migration %d failed", int32(tx.ID))
 	}
 
-	state.Set(migrationKey(tx.ID), msg.Data)
+	state.Set(migrationKey(tx.ID), []byte{1})
```

Comment:
Why is this changed? If you want to change this to save some space, a feature flag is needed or @enlight approve this.

Reply:
yep @enlight approved 
https://github.com/loomnetwork/loomchain/pull/1431#pullrequestreview-278234687

Reply:
Approved with a feature flag lol, so you need to wrap this with a feature flag to prevent consensus failure.

Reply:
Since migration txs can have input parameters now it doesn't make much sense to store all that in the app store, we only use this key to check if a migration has already ran, the value we store here is irrelevant.

This change does need a new feature flag.